src/gas-price/gas-price.ts

```diff
@@ -187,10 +190,19 @@ export const getAirseekerRecommendedGasPrice = async (
   const hasSufficientSanitizationData = gasPrices.some((gasPrice) => gasPrice.timestampMs <= minTimestampMs);
 
   // Check if the multiplied gas price is within the percentile and return the smaller value
-  const sanitizedGasPrice =
-    hasSufficientSanitizationData && percentileGasPrice && gasPrice.gt(percentileGasPrice)
-      ? percentileGasPrice
-      : gasPrice;
+  if (!hasSufficientSanitizationData && percentileGasPrice && gasPrice.gt(percentileGasPrice)) {
+    logger.warn('Gas price could be sanitized but there is not enough historical data');
+  }
+  let gasPriceToUse: ethers.BigNumber;
+  if (hasSufficientSanitizationData && percentileGasPrice && gasPrice.gt(percentileGasPrice)) {
+    logger.warn('Sanitizing gas price', {
+      gasPrice: gasPrice.toString(),
+      percentileGasPrice: percentileGasPrice.toString(),
+    });
+    gasPriceToUse = percentileGasPrice;
+  } else {
+    gasPriceToUse = gasPrice;
+  }
```

Comment:
```suggestion
  if (!hasSufficientSanitizationData && percentileGasPrice && gasPrice.gt(percentileGasPrice)) {
    logger.warn('Gas price could be sanitized but there is not enough historical data');
  }

  if (hasSufficientSanitizationData && percentileGasPrice && gasPrice.gt(percentileGasPrice)) {
    logger.warn('Sanitizing gas price', {
      gasPrice: gasPrice.toString(),
      percentileGasPrice: percentileGasPrice.toString(),
    });
    return multiplyBigNumber(percentileGasPrice, recommendedGasPriceMultiplier);
  } 
  
  return multiplyBigNumber(gasPrice, recommendedGasPriceMultiplier);
```
but super super minor/matter of opinion.